packages/library-base/src/commonMain/kotlin/io/realm/Realm.kt

```diff
@@ -27,7 +29,7 @@ import kotlin.reflect.KClass
  *
  * @see Configuration
  */
-interface Realm : TypedRealm {
+interface Realm : TypedRealm, Flowable<RealmChange<Realm>> {
```

Comment:
I don't think we should let internal interfaces leak into the public API. Should this have been on `RealmImpl` instead?